perfkitbenchmarker/packages/fio.py

```diff
@@ -153,4 +153,23 @@ def ParseResults(job_file, fio_json_result):
             sample.Sample('%s:latency' % metric_name,
                           job[mode]['lat']['mean'],
```

Comment:
the parsing here is not entirely correct: we should get 'clat' rather than 'lat'

and the unit is not always usec, some times it's msec, fio changes the unit dynamically based on the outcome.